### DISCUSSION TITLE
Happy holidays!

### DISCUSSION THREAD

**Discussion by Taitava** [Taitava/obsidian-shellcommands#131] [Offtopic]
Happy holidays to all! ❄️ 

I'll spend about one week without GitHub and coding. See you! 😃

**Comment by FelipeRearden** [comment]
Hello @

Happy New Year !!!! 

I'm sorry for my absence, I have been sick since Christmas (I sent you a DM on the forum than we can talk privately)

I hope you had a good start of 2022 !
